modules/about-rhde.adoc

```diff
@@ -46,12 +46,12 @@ The latest release notes for each product that is part of {product-title} are av
 ^|9.4
 ^|4.16
 ^|Generally Available
-^|4.16.0&#8594;4.16.z and 4.16&#8594;maximum two minor versions
+^|4.16.0&#8594;4.16.z 4.14&#8594;4.16 and 4.15&#8594;4.16
```

Comment:
This is not correct. You cannot use 4.14 with RHEL 9.4 to my knowledge? @DanielFroehlich is there a reason we removed the footnote about updating RHEL? This table row now implies that MicroShift 4.14 is supported on RHEL 9.4. It also removes the doc update for the Y+2 version feature, see https://github.com/openshift/openshift-docs/pull/73988/files#diff-e6fe883fea97caca8450bee979ffeb8a26e030bc4b2d183b373b4424fd4a3c8bR22. Just want to be sure these changes are what you intend.

Reply:
The goal was:
* get rid of the footnote, which makes thinks more complicated
* remove the Y+2 promise into the future. we should not make any promises we are not 100% sure we can fullfill, but only document on what is supported.

But I get your point that one could read the 4.14 row as one could update to 4.16 but on RHEL 9.2, which is indeed not supported.
How about we change the  rightmost column from "MicroShift supported updates" to "Supported updates from **previous** MicroShift versions", and then remove all the "future" entries we have, e.g. in the V4.14 row, remove the V4.14->4.15. Like we currently have in the V4.16 row.
That would simplify the table, and make clear that V4.16 is only supported and requires an OS upgrade. If we want to make clear, we could add a "*" on each update that actually requires an OS update and describe that in a footnote.
WDYT?

Reply:
* OK, no future promises. I think we want to show 4.14>4.16, though, since that is a 4.16 feature (and this will show with the 4.16 docs). 
* OK to change the MicroShift column title, how about "Supported MicroShift>MicroShift updates" and we can use the arrow there to keep it short?
* The docs sites don't support unnumbered footnotes, unfortunately. Which means multiple footnotes if we note each MicroShift update that needs an OS update. What if we just mention that an OS update might be needed up in line 37? 
* I think we should retain updates that span the same OS are are GA, such as 4.14>4.15.

Reply:
@DanielFroehlich should I make the suggested changes? I think the suggestions make sense.

Reply:
@ShaunaDiaz and @DanielFroehlich Preview is ready